templates/upload/index.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+
+    <div>
+        {% if import_result is defined %}
+            {% if import_result is not null %}
+                <p><span class="text-bold">Nombre de personnes</span> : <span>{{ import_result.countPersons}}</span></p>
+                <p><span class="text-bold">Nombre de projets</span> : <span>{{ import_result.countProjects}}</span></p>
+                <p><span class="text-bold">Nombre de donations</span> : <span>{{ import_result.countDonations}}</span></p>
+                <p><span class="text-bold">Nombre de récompenses</span> : <span>{{ import_result.countRewards}}</span></p>
+                <p><span class="text-bold">Erreurs : </span> : <span>{{ import_result.errorsString}}</span></p>
```

Comment:
I think a table with 3 col should be done here.
| Line | Property | Error message |
| ------ | ----------- | -------------------- |
| 42    | firstName | this value should not be bkank |
| a... | a... | a... |